### PR TITLE
fix(meta-preview): update meta preview images and description

### PR DESCRIPTION
**Pull request by davidholyko**
#### Summary

- Work Type: Fix
- Fixes: website preview

#### What I did

- fixes needing to overwrite custom domain every deploy
- adds content preview
- update deploy steps

#### Notes / Research / References

- https://github.com/tschaub/gh-pages/issues/213
-

#### Screenshots

After:
Screen Shot 2021-02-22 at 12 37 55 PM

#### Checklist

- [x] Commits are descriptive
- [x] Has Summary

### CODE REVIEW

File: webpack.config.js [hunk 1]
```diff
@@ -34,6 +34,16 @@ const plugins = [
         from: 'public/manifest.json',
         to: 'assets/manifest.json',
         toType: 'file'
+      },
+      {
+        from: 'public/medithanks.png',
+        to: 'assets/medithanks.png',
+        toType: 'file'
+      },
+      {
```
Comment (davidholyko) [inline webpack.config.js:43]: this fixes needing to update custom domain url after deploy

**Review by simonchen92 — APPROVED** [review]
LGTM

File: webpack.config.js [hunk 2]
```diff
@@ -34,6 +34,21 @@ const plugins = [
...
+        from: 'public/medithanks-preview.png',
+        to: 'assets/medithanks-preview.png',
+        toType: 'file'
+      },
+      {
+        from: 'CNAME',
```
Comment (davidholyko) [inline webpack.config.js:49]: this fixes needing to update custom domain url after deploy

File: package.json [hunk 3]
```diff
@@ -42,7 +42,7 @@
     "eject": "react-scripts eject",
     "prettier-format": "prettier --config .prettierrc 'src/**/*.{js,scss}' --write",
     "lint": "eslint . --ext .js",
-    "deploy": "gh-pages --dist dist"
+    "deploy": "rm -rf dist && npm run build && gh-pages --dist dist"
```
Comment (davidholyko) [inline package.json:45]: deletes old dist directory if already exists so we don't accidentally just deploy the previous version

**Pull request merged**